runtime/domain.c

```diff
@@ -1754,3 +1767,37 @@ CAMLprim value caml_domain_dls_get(value unused)
   CAMLnoalloc;
   return Caml_state->dls_root;
 }
+
+CAMLprim value caml_recommended_domain_count(value unused)
+{
+  CAMLnoalloc;
+  long n = -1;
```

Comment:
Not that we'll see a machine where it matters, but 🙂
```suggestion
  intnat n = -1;
```